src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -583,3 +584,82 @@
              [:dispatch
               [:navigate-to-within-stack
                [:screen/wallet.swap-select-asset-to-pay :screen/wallet.swap-select-account]]]])})))
+
+(rf/reg-event-fx :wallet/get-swap-proposal-fee
```

Comment:
This is a simplified version of `:wallet/start-get-swap-proposal`, designed to fetch only the relevant fee fields early using a synchronous RPC call.